WordPress/Classes/ViewRelated/Cells/InlineEditableNameValueCell.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+import WordPressAuthenticator
+
+class InlineEditableNameValueCell: WPTableViewCell {
+
+    @IBOutlet weak var nameValueWidthRatioConstraint: NSLayoutConstraint!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var valueTextField: LoginTextField!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        selectionStyle = .none
+        nameLabel.textColor = WPStyleGuide.darkGrey()
+        nameLabel.font = WPStyleGuide.tableviewTextFont()
+        nameLabel.numberOfLines = 0
+        valueTextField.textColor = WPStyleGuide.greyDarken10()
+        valueTextField.font = WPStyleGuide.tableviewTextFont()
+        valueTextField.borderStyle = .none
+        valueTextField.contentInsets = WPStyleGuide.edgeInsetForLoginTextFields()
+        // swiftlint:disable:next inverse_text_alignment
```

Comment:
I think this might not wok as expected in Right-To-Left languages. There's a (admittedly, not the most beautiful) pattern to handle this in https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Activity/ActivityDetailViewController.swift#L123-132 — maybe it could be useful here :)?

Reply:
Done

Reply:
I tested the new commit with NSForceRightToLeftWritingDirection= YES and it worked all fine by the way